launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml

```diff
@@ -65,3 +65,9 @@
 
     # regularization scale factor
     regularization_scale_factor: 0.01
+
+    # whether use de-grounded pointcloud calculate score
+    use_no_ground_pointcloud: true
```

Comment:
`use_no_ground_pointcloud` may be tricky for beginners. (For example they might take it as a flag for using degrounded scans for NDT scan matching)
How about something like as follow?
- `estimate_scores_for_degrounded_scan`
- `calculate_degrounded_scan_matching_score`